apps/demos/Demos/Scheduler/ToolbarCustomization/React/App.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react';
+
+import Scheduler, { Toolbar, Item, Resource, type SchedulerTypes, type SchedulerRef } from 'devextreme-react/scheduler';
+import SelectBox, { type SelectBoxTypes } from 'devextreme-react/select-box';
+
+import { assignees, schedulerDataSource, currentDate } from './data.ts';
+
+const views: SchedulerTypes.ViewType[] = ['day', 'week', 'workWeek', 'month'];
+const selectBoxPlaceholder = 'Select Employee';
+const inputAttr = { 'aria-label': selectBoxPlaceholder };
+
+const App = () => {
+  const schedulerRef = useRef<SchedulerRef>(null);
+  const [assigneesFilterValue, setAssigneesFilterValue] = useState<number>();
+
+  const onAssigneesFilterChange = useCallback((event: SelectBoxTypes.ValueChangedEvent) => {
+    const scheduler = schedulerRef.current!.instance()!;
+    const filter = event.value ? ['assigneeId', 'contains', event.value] : null;
+
+    schedulerDataSource.filter(filter);
+    scheduler.option('dataSource', schedulerDataSource);
+    setAssigneesFilterValue(event.value);
+  }, []);
+  const toggleButtonOptions = useMemo(() => ({
```

Comment:
Let's add empty line here

```suggestion
  }, []);
  
  const toggleButtonOptions = useMemo(() => ({
```